enhancements/storage/vsphere-problem-detector.md

```diff
@@ -0,0 +1,230 @@
+---
+title: vsphere-problem-detector
+authors:
+  - "@jsafrane"
+reviewers:
+  - "@hekumar"
+  - "@jspeed"
+  - "@jcpowermac"
+  - "@staebler"
```

Comment:
Should these not be GitHub usernames? I was under the impression that was the convention at least

Reply:
They probably should...